Enhances Team Admin member role view.

jSylvestre opened the pull request:
Improves the Team Admin member role view by adding styling and pagination.

The table displaying members and their roles is enhanced with data table functionality, including pagination, sorting, and responsive styling, to improve usability.

Adds styles and scripts report partials.

jSylvestre committed:
Enhances Team Admin member role view.

Improves the Team Admin member role view by adding styling and pagination.

The table displaying members and their roles is enhanced with data table functionality, including pagination, sorting, and responsive styling, to improve usability.

Adds styles and scripts report partials.

jSylvestre committed:
rebuild

jSylvestre committed:
Updates the Azure Pipelines VM image

Updates the Azure Pipelines VM image from 'windows-2019' to 'windows-latest'.

This ensures the pipeline uses the most up-to-date Windows environment, providing access to the latest features, security patches, and tooling.

jSylvestre committed:
Updates CI pipeline and Node version

Updates the CI pipeline configuration to use Node 20.x.

This change ensures compatibility with the latest dependencies and
improves the build process.
The pipeline now includes steps to build, test, and publish the
application, including Keas.Mvc and related job projects.

jSylvestre committed:
update readme

jSylvestre commented:
the formatting changes in the yml file happened when I saved in vs code.

sprucely commented on Keas.Mvc/ClientApp/package.json:
```diff
@@ -95,6 +95,6 @@
   },
   "engines": {
```
Changes in npm and node usually go together. Should probably update the npm version here to whatever comes with node 20.0.0.

sprucely reviewed (changes requested):
The YML looks okay. I ran it by copilot, and it says the whitespace change doesn't impact semantic meaning as long as it is consistent.

jSylvestre committed:
Updates npm engine version

Updates the minimum required npm version to 10.0.0.

This ensures compatibility with newer dependencies and build tools,
as well as aligns with the project's ongoing upgrades.

jSylvestre commented on Keas.Mvc/ClientApp/package.json:
```diff
@@ -94,7 +94,7 @@
   "engines": {
-    "npm": ">=9.5.1",
-    "node": ">=18.16.0"
+    "npm": ">=10.0.0",
```
This is the one AI and google appear to say is the correct one.
